### PR TITLE
Capstone Project Creationism vs Evolution Global Summit

### PR DESCRIPTION
**Pull request by oovillagran**
**HTML/CSS & JavaScript capstone project - Conference page**

This capstone project intends to show the acquired abilities in the Portal Module of the Microverse Full Stack Web Developer program.

I create a conference page using the guidelines provided by [Common Creatives](https://creativecommons.org/licenses/by-nc/4.0/) and the original designer [Cindy Shin](https://www.behance.net/adagio07).  In this project you will find a Home Page and About Page according to the project requirements.

Following you can see the project requirements:

**General requirements**
- [No linter errors](https://github.com/microverseinc/linters-config).
- Correct use of [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Work documented [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Follow Microverse's list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Follow Microvers's list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

**Project requirements**
- [ ] I personalized the content of my page. The topic is "Creation vs Evolution Global Summit 2023".

- [ ] I followed these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

     -  Colors.
     - Typography: font face, size and weight, I used [Lato](https://www.latofonts.com/).
     - Layout: composition and space between elements.
- [ ] The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

- [ ] You can use a CSS framework (for example Bootstrap) for styling, if you want to. *I used Boostrap*.

- [ ] You should build only these 2 pages:

     - The home page.
     - The about page.
- [ ] Each of these pages should have versions for 2 different screen sizes:

     - Mobile: up to 768px wide.
     - Desktop: 768px or wider.
- [ ] Interactions

     - Links
          - The home page should have a link in the menu to the about page.
          - The logo in the header links to the home page.
     - Mobile menu
          - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
          - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
          - The mobile menu should have a close (X) button that closes the menu.

- [ ] Dynamic page

     - The section "Featured speakers" should be created dynamically in JavaScript.
     - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,929 @@
...
+    background-size: contain;
+    border: #fff 1px solid;
+    position: relative;
+  }
+
+  .intro {
+    margin: 6rem 0 0 12rem;
+    padding: 0 0 5rem;
+    width: 70%;
+  }
```
Comment (omarsalem7) [inline styles.css:518]: - [x] Kindly fix the background for the intro section as in the following image
>Screenshot 2023-03-23 235646

File: about.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  main.classList.toggle('active');
+  footer.classList.toggle('active');
+});
+
+document.querySelectorAll('.link-menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  main.classList.remove('active');
+  footer.classList.remove('active');
+}));
```
Comment (omarsalem7) [inline about.js]: - [x] Kindly consider when the mobile menu is opened make the menu does not scroll to be fixed on screen, you can look at the following image to get what I mean
>Recording 2023-03-24 at 00 01 12

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @oovillagran 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Good presentation demo ✔️
- Live demo works properly✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.js (hunk 2 above)
Comment (oovillagran) [reply about.js]: Thank you @omarsalem7 I will make changes according to your suggestions.

**Review by ouasamine — APPROVED** [review]
Hi @oovillagran  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Good communication with the reviewer ✔️ 
- Nice design 👏🏽 👏🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thanks @ouasamine !!!👌👏👍

**Comment by oovillagran** [comment]
Thanks @ouasamine !!!👌👏👍

**Pull request merged**